test/hw8/test_hw8_2_sql_task.py

```diff
@@ -11,40 +11,33 @@
 def database() -> None:
     con = sqlite3.connect("db_test.sqlite")
     cur = con.cursor()
-    cur.execute(
-        "CREATE TABLE presidents(name VARCHAR(15), val_1 INT, val_2 VARCHAR(20));"
-    )
-    cur.execute(
-        "INSERT INTO presidents(name, val_1, val_2) VALUES ('Big Man Tyrone', 101, 'Kekistan');"
-    )
-    cur.execute(
-        "INSERT INTO presidents(name, val_1, val_2) VALUES ('Yeltsin', 999, 'Russia');"
-    )
-    cur.execute(
-        "INSERT INTO presidents(name, val_1, val_2) VALUES ('Trump', 1337, 'US');"
-    )
+    with open("test/hw8/SQL_commands.txt") as f:
+        cur.execute(f.readline().strip())
```

Comment:
And now we can run the whole command without splitting (with only one execution)

Reply:
Also for files like that we may use .sql extension